src/Symfony/Component/Cache/Tests/Adapter/MemcachedAdapterTest.php

```diff
@@ -174,33 +174,32 @@ public static function provideServersSetting(): iterable
     }
 
     /**
-     * @dataProvider provideDsnWithOptions
+     * @requires extension memcached
      */
-    public function testDsnWithOptions(string $dsn, array $options, array $expectedOptions)
+    public function testOptionsFromDsnWinOverAdditionallyPassedOptions()
     {
-        $client = MemcachedAdapter::createConnection($dsn, $options);
+        $client = MemcachedAdapter::createConnection('memcached://localhost:11222?retry_timeout=10', [
+            \Memcached::OPT_RETRY_TIMEOUT => 8,
+        ]);
 
+        $this->assertSame(10, $client->getOption(\Memcached::OPT_RETRY_TIMEOUT));
         foreach ($expectedOptions as $option => $expect) {
             $this->assertSame($expect, $client->getOption($option));
         }
```

Comment:
```suggestion
```
These lines can be removed since the variable $expectedOptions is not set anymore.

Reply:
oops 🙈